src/SprykerEco/Zed/PunchoutCatalogs/Communication/Form/ConnectionSetupSubForms/PunchoutCatalogConnectionCartForm.php

```diff
@@ -127,6 +128,7 @@ protected function addTotalsModeField(FormBuilderInterface $builder)
         $builder->add(PunchoutCatalogConnectionCartTransfer::TOTALS_MODE, ChoiceType::class, [
             'label' => 'Totals Mode',
             'choices' => [
+                'Disabled' => 'disabled',
```

Comment:
translations are missing, but must be provided
please adjust `data/translation/Zed/en_US.csv` and `data/translation/Zed/de_DE.csv` accordingly

Reply:
@DmitryLymarenko added.